support-files/mini-benchmark.sh

```diff
@@ -156,28 +207,23 @@ mariadb -e "
   CREATE USER IF NOT EXISTS sbtest@localhost;
   GRANT ALL PRIVILEGES ON sbtest.* TO sbtest@localhost"
 
-sysbench oltp_read_write prepare --tables=20 --table-size=100000 | tee sysbench-prepare.log
+sysbench "$WORKLOAD" prepare --tables=20 --table-size=100000 | tee sysbench-prepare.log
 sync && sleep 1 # Ensure writes were propagated to disk
 
-# Save results of this run in a subdirectory so that they are not overwritten by
-# the next run
-TIMESTAMP="$(date -Iseconds)"
-mkdir "mini-benchmark-$TIMESTAMP"
-cd "mini-benchmark-$TIMESTAMP" || exit 1
-
 # Run benchmark with increasing thread counts. The MariaDB Server will be using
 # around 300 MB of RAM and mostly reading and writing in RAM, so I/O usage is
 # also low. The benchmark will most likely be CPU bound to due to the load
 # profile, and also guaranteed to be CPU bound because of being limited to a
 # single CPU with 'tasksel'.
-for t in 1 2 4 8 16
+for t in $THREADS
 do
   # Prepend command with perf if defined
-  # Output stderr to stdout as perf outpus everything in stderr
-  $PERF $TASKSET_SYSBENCH sysbench oltp_read_write run --threads=$t --time=60 --report-interval=10 2>&1 | tee sysbench-run-$t.log
+  # Output stderr to stdout as perf outputs everything in stderr
+  # shellcheck disable=SC2086
```

Comment:
`$t` and `$DURATION` could still be quoted. 

Optional: To eliminate the shell check warning `THREADS`, `PERF` and `TASK_SETSYSBENCH` could all be changed to arrays.

Reply:
It _might_ be worth leaving `$t` and `$DURATION` unquoted for readability/consistency with the rest of the line; if either of the values were globbed or word split then that would mean the user supplied bad inputs. Either way (quoting/unquoting) `sysbench` would complain that the values are in an unrecognized format.

If we agree that the values (`$t`, `$DURATION`) should be left unquoted, that would mean `shellcheck disable=SC2086` would remain, regardless of converting the other arguments into arrays ([`SC2086`](https://www.shellcheck.net/wiki/SC2086) complains about _all_ the arguments on the line being unquoted).